### DISCUSSION TITLE
Understanding How Depreciation Works with Percentage in Snipe-IT

### DISCUSSION THREAD

**Discussion by FirstUnionGroup** [snipe/snipe-it#12468] [Q&A]
I have read that the depreciation rate for computers is 40%. For Snipe-IT, we're given 3 fields: Depreciation Name, Number of Months and Minimum Value after Depreciation. Can someone please explain to me How I would be able to account for the 40% depreciation of the computer assets? I am not seeing any documentation of how to utilize what's available for depreciation.

**Comment by tpaljor2010** [comment]
I have the same dilemma. Any words from the team is highly appreciated.
